sdk/cosmos/azure_data_cosmos/src/clients/cosmos_client.rs

```diff
@@ -96,6 +98,34 @@ impl CosmosClient {
         })
     }
 
+    #[allow(dead_code)]
+    async fn initialize(client: CosmosClient) -> azure_core::Result<Self> {
+        let _account = client.get_database_account(None).await?.into_body()?;
+        Ok(client)
+    }
+
+    /// Retrieves the Cosmos DB account ("database account") properties.
+    ///
+    /// # Arguments
+    /// * `options` - Optional request options (currently unused for custom
+    ///   headers, but the context can carry per-call metadata for tracing or
+    ///   cancellation).
+    pub async fn get_database_account(
```

Comment:
Should this be `pub` or just `pub(crate)`? Is it an API we'd want to expose to users?

Reply:
Removed the get_database_account call for the time being. I will introduce this later with the `global_endpoint_manager` PR.